spec/features/create_destroy_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe "Create and Destroy Comment" do
+
+  describe "successful" do
+    it "Signs the user up, in, makes a list, then adds and deletes an Item through BDD" do
```

Comment:
The `it` description can be more of a summary of the main feature under test. Leave the precise steps taken to be described by the code already in this test.